src/kvstore/raftex/RaftPart.cpp

```diff
@@ -330,6 +329,23 @@ AppendLogResult RaftPart::canAppendLogs() {
     return AppendLogResult::SUCCEEDED;
 }
 
+void RaftPart::addLearner(const HostAddr& addr) {
+    CHECK(!raftLock_.try_lock());
+    if (addr == addr_) {
+        LOG(INFO) << idStr_ << "I am learner!";
+        return;
+    }
+    auto it = std::find_if(hosts_.begin(), hosts_.end(), [&addr] (const auto& h) {
+                return h->address() == addr;
+            });
+    if (it == hosts_.end()) {
+        hosts_.emplace_back(std::make_shared<Host>(addr, shared_from_this(), true));
+        LOG(INFO) << idStr_ << "Add learner " << addr;
+    } else {
+        LOG(INFO) << idStr_ << "The host " << addr << " has been existed as "
```

Comment:
why we can ignore the conflict?

Reply:
What's your plan?

Reply:
Just a question, because I do not has the context that how to use learner